prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -4,7 +4,7 @@ elements:
     hl7Spec: [ '%{hl7OBXField}-1' ]
 
   - name: result-value-type-st
-    condition: '%resource.value is String or %resource.value.exists().not()'
```

Comment:
This condition is incorrect as a cancelled observation will not have this set but can have value populate.